### ISSUE TITLE
Code formatting issue

### ISSUE THREAD

**Issue by SOF3** [Falkirks/blog#1]
I didn't explore into which library you're using to highlight your code, but it seems like your PHP code isn't highlighted properly because the highlighter expects your code to be _full_ and _valid_ PHP, with a `<?php` tag.

See http://blog.falkirks.com/setting-huge-areas-of-blocks for example.

**Comment by falkirks** [comment]
Yeah, I just imported from Ghost without checking anything. I will look at that when I get a chance.
